src/index.html

```diff
@@ -9,6 +9,62 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div
+      class="img">
```

Comment:
```suggestion
      <div class="img">
```
If you have only one attribute it doesn`t make sense to move it to new line, change it everywhere in similar cases